cloudinit/sources/__init__.py

```diff
@@ -83,6 +83,19 @@ def __str__(self) -> str:
         return self.value
 
 
+class NicOrder(Enum):
+    """
+    Represents the canonical list of network config sources that cloud-init
```

Comment:
I think `network config sources` could be misleading, as cloud-init allows users to define network configurations, see https://docs.cloud-init.io/en/latest/reference/network-config.html#network-configuration-sources .

What about just the following?

```
    """Represents ways to sort NICs"""
```

Reply:
I have already fix it ,thanks.